app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
```

Comment:
You don't have to import all these classes

Reply:
Probably than your annotation won't work. But I'm not sure that 3 imports worth it